packages/core/admin/admin/src/features/Configuration.tsx

```diff
@@ -76,9 +76,7 @@ const ConfigurationProvider = ({
   );
   const token = useAuth('ConfigurationProvider', (state) => state.token);
 
-  const {
-    allowedActions: { canRead },
-  } = useRBAC(permissions);
+  const { allowedActions } = useRBAC(permissions);
```

Comment:
I haven't tested the fix but this looks like the only change is you made is to no longer destructure canRead right? I don't see how that would change the runtime behavior